tests/virt_autotest/login_console.pm

```diff
@@ -70,8 +70,26 @@ sub login_to_console {
     save_screenshot;
     send_key 'ret';
 
-    assert_screen(['linux-login', 'virttest-displaymanager'], $timeout);
+    my $host_installed_version = get_var('VERSION_TO_INSTALL', get_var('VERSION', ''));
+    ($host_installed_version) = $host_installed_version =~ /^(\d+)/;
+    if (check_var('offline_upgrade', 'yes') && ($host_installed_version eq '11')) {
+        save_screenshot;
+        assert_screen('sshd-server-started-config', 180);
+        save_screenshot;
+        use_ssh_serial_console;
+        save_screenshot;
+        #start system first configuration after finishing upgrading from sles-11-sp4
+        type_string("yast.ssh\n");
+        save_screenshot;
+        wait_still_screen('30', $timeout);
```

Comment:
Again, you pass a string, not a number. Please change that. Also, you ask for a stilltime of 30s. That is: The screen must stay unchanged for 30s. I do not see any reason for that.

Reply:
> Again, you pass a string, not a number. Please change that. Also, you ask for a stilltime of 30s. That is: The screen must stay unchanged for 30s. I do not see any reason for that.

1) Which string do you refer to ? '30' ?  Should it be just 30 without any quotes ?   
2) It is reasonable. I can not just jump back to sol console directly after issuing "yast.ssh". The ongoing configuration operation should be recorded. So wait_still_screen here is used as a mechanism to make sure configuration is done and nothing else is undertaken. Maybe you can give it a different value, for example, 15s. But that does not make any big difference. 30s is more stable and robust.

Reply:
1. Yes, we can just pass a number, meaning 30 without quotes
2. I'm not a fan of magic numbers anywhere. Syncing on a still screen is not a good approach in general, so the question is actually if we can get something better. E.g. wait for some string on serial. Using `wait_still_screen` in places like this just shows that our tooling has to be improved, otherwise we will have such hacks in the test code.

Reply:
New changes had been committed. assert_screen with new needle 'will-linux-login'. This needle looks like:
![image](https://user-images.githubusercontent.com/38779103/46052735-38e89800-c172-11e8-9b7a-b435a881204e.png)

Reply:
Please help merge if nothing else.